packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -31,20 +31,22 @@ def test_if_docker_app_can_be_deployed(dcos_api_session):
     Verifies that a marathon app inside of a docker daemon container can be
     deployed and accessed as expected.
     """
-    dcos_api_session.marathon.deploy_test_app_and_check(*get_test_app_in_docker(ip_per_container=False))
+    dcos_api_session.marathon.deploy_test_app_and_check(
+        *get_test_app(network=Network.BRIDGE, container_type=Container.DOCKER, container_port=9080))
 
 
-@pytest.mark.parametrize("healthcheck", [
-    "HTTP",
-    "MESOS_HTTP",
+@pytest.mark.parametrize('healthcheck', [
```

Comment:
That's how it was before so this will remain outside of scope in this case, but the next time this code gets touched this marginal style  error can be addressed if anyone cares to notice